controllers/memberstatus/memberstatus_controller_test.go

```diff
@@ -113,7 +119,7 @@ func TestOverallStatusCondition(t *testing.T) {
 		memberOperatorDeployment := newMemberDeploymentWithConditions(status.DeploymentAvailableCondition(), status.DeploymentProgressingCondition())
 		memberStatus := newMemberStatus()
 		getHostClusterFunc := newGetHostClusterReady
-		reconciler, req, fakeClient := prepareReconcile(t, requestName, getHostClusterFunc, allNamespacesCl, append(nodeAndMetrics, memberOperatorDeployment, memberStatus)...)
+		reconciler, req, fakeClient := prepareReconcile(t, requestName, getHostClusterFunc, allNamespacesCl, mockLastGitHubAPICall, test.MockGitHubClientForRepositoryCommits(buildCommitSHA, time.Now().Add(-time.Hour*1)), append(nodeAndMetrics, memberOperatorDeployment, memberStatus)...)
 
 		// when
 		res, err := reconciler.Reconcile(context.TODO(), req)
```

Comment:
for these happy paths `t.Run("All components ready", func(t *testing.T) {` shouldn't we update also the assertion so it expects the resource revision to be set? Just asking :wink:

Reply:
You are right, I've missed those, I've now added the expected disabled condition. I was thinking that maybe at some point I can remove the specific test that verifies if the component is up to date _"member operator deployment version is up to date"_ and have this check in _"All components ready"_. For now I did it the same way as host-operator, but probably is worth refactoring it.

Plz see [78a30e1](https://github.com/codeready-toolchain/member-operator/pull/446/commits/78a30e10315b864af392e7a11c435de06468920a) and [daee1ad](https://github.com/codeready-toolchain/member-operator/pull/446/commits/daee1ad8962b8530c1da5e096e0f3ac12a8b7a1d)